rhesus-monkeys/scripts/convert-meta.py

```diff
@@ -0,0 +1,97 @@
+import polars as pl
+import re
+from pathlib import Path
+
+# Set the folder path for raw-data
+resource_dir = Path(__file__).resolve().parent.parent
+folder_path = resource_dir / "data-raw"
+
+# Create the infant metadata files
+    # FIO There is one extra infant in 07, all infants in 08 are included in 07
+df07 = pl.read_csv(folder_path / "data07.csv")
+df08 = pl.read_csv(folder_path / "data08.csv")
+
+df_infant = df07.select(pl.col("Infant_ID"), pl.col("Mother_ID"), pl.col("GD_Delivery").alias("gestation_day_delivery")
+, pl.col("Fostered"), pl.col("Foster_ID"))
```

Comment:
```suggestion
df_infant = (
		df07.select(["Infant_ID", "Mother_ID", "GD_Delivery", "Fostered", "Foster_ID"])
		.rename({"GD_Deliver": "gestation_day_delivery"})
)
```

You can give multiple columns with `[]`. Likewise, you can use "method chaining" to have multiple functions continue across lines. I use `rename()` here to make it easier to select all columns and then rename the ones I want (rather than use `.alias()`).